test/alternator/test_tracing.py

```diff
@@ -229,32 +239,57 @@ def test_tracing_all(with_tracing, test_table_s_isolation_always, dynamodb):
+            test_table_s_isolation_always_2.name: [{'PutRequest': {'Item': {'p': p_batchwriteitem2}}}],
+        })
+        table.meta.client.batch_get_item(RequestItems = {
+            test_table_s_isolation_always.name: {'Keys': [{'p': p_batchgetitem1}]},
+            test_table_s_isolation_always_2.name: {'Keys': [{'p': p_batchgetitem2}]},
+        })
 
     # Check the traces. NOTE: the following checks are fairly arbitrary, and
     # may break in the future if we change the tracing messages...
-    expect_tracing_events(dynamodb, p_putitem, ['PutItem', 'CAS successful'])
-    expect_tracing_events(dynamodb, p_getitem, ['GetItem', 'Querying is done'])
-    expect_tracing_events(dynamodb, p_deleteitem, ['DeleteItem', 'CAS successful'])
-    expect_tracing_events(dynamodb, p_updateitem, ['UpdateItem', 'CAS successful'])
-    expect_tracing_events(dynamodb, p_batchgetitem, ['BatchGetItem', 'Querying is done'])
-    expect_tracing_events(dynamodb, p_batchwriteitem, ['BatchWriteItem', 'CAS successful'])
-    expect_tracing_events(dynamodb, p_query, ['Query', 'Querying is done'])
-    expect_tracing_events(dynamodb, p_scan, ['Scan', 'Performing a database query'])
+    expect_tracing_events(dynamodb, p_putitem, ['PutItem', 'CAS successful'], table.name)
+    expect_tracing_events(dynamodb, p_getitem, ['GetItem', 'Querying is done'], table.name)
+    expect_tracing_events(dynamodb, p_deleteitem, ['DeleteItem', 'CAS successful'], table.name)
+    expect_tracing_events(dynamodb, p_updateitem, ['UpdateItem', 'CAS successful'], table.name)
+    expect_tracing_events(dynamodb, p_batchgetitem, ['BatchGetItem', 'Querying is done'], table.name)
+    expect_tracing_events(dynamodb, p_batchwriteitem, ['BatchWriteItem', 'CAS successful'], table.name)
+    expect_tracing_events(dynamodb, p_query, ['Query', 'Querying is done'], table.name)
+    expect_tracing_events(dynamodb, p_scan, ['Scan', 'Performing a database query'], table.name)
 
-    op = 'Alternator PutItem'
+    op_1 = 'Alternator PutItem'
+    op_2 = 'Alternator BatchWriteItem'
     found_truncated = False
     found_not_truncated = False
-    for entry in iterate_tracing_sessions(dynamodb):
-        if entry['request'] == op:
-            p = entry['parameters']
+    found_double_tables = False
+    for entry in iterate_tracing_sessions(dynamodb, table.name):
+        p = entry['parameters']
+        if entry['request'] == op_1:
             if beginning_of_p_putitem_really_long in p:
                 if '<truncated>' in p:
                     found_truncated = True
                 else:
                     found_not_truncated = True
-                if found_truncated and found_not_truncated:
-                    break
-    assert found_truncated and found_not_truncated
+        elif entry['request'] == op_2:
+            if p_batchwriteitem1 in p and p_batchwriteitem2 in p:
+                tables = entry['parameters']
```

Comment:
Above you already set p = entry['parameters'], so why do you want to set "tables" to the same thing?